content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -342,21 +341,9 @@ export const ConceptPage: NextPage<Props> = ({
           <TypeLabel>{conceptTypeDisplayName(conceptResponse)}</TypeLabel>
           <Space v={{ size: 's', properties: ['margin-top', 'margin-bottom'] }}>
             <HeroTitle>{conceptResponse.label}</HeroTitle>
-            <ConceptDescription className={font('intr', 5)}>
-              <BetaMessage
-                message={
-                  <>
-                    We&rsquo;re working to improve the information on this page.
-                    You can <Link href="/user-panel">join our user panel</Link>{' '}
-                    or{' '}
-                    <Link href="https://roadmap.wellcomecollection.org/">
-                      submit an idea
-                    </Link>{' '}
-                    that would help you use our website.
-                  </>
-                }
-              />
-            </ConceptDescription>
+            <ConceptDescription
```

Comment:
I think this (and the related styles) can be removed too

Reply:
Ah yes - thanks!!